pkg/cloudprovider/providers/azure/azure_blobDiskController.go

```diff
@@ -79,52 +79,55 @@ func newBlobDiskController(common *controllerCommon) (*BlobDiskController, error
 	return &c, nil
 }
 
-// CreateVolume creates a VHD blob in a given storage account, will create the given storage account if it does not exist in current resource group
```

Comment:
cannot comment on line 64, but it is a mess there. use a var block.

Reply:
fixed